1-prep/2-chain-spec.md

```diff
@@ -85,34 +85,34 @@ spec file ccan be reproduced. The second part is obtained similarly with `//Bob`
 Polkadot address for `//Alice//stash` (`sr25519` cryptography).
 
 ```bash
-$ subkey inspect --scheme sr25519 --network polkadot //Alice//stash
+$ subkey inspect --scheme sr25519 --network substrate //Alice//stash
 Secret Key URI `//Alice//stash` is account:
   Secret seed:      0x3c881bc4d45926680c64a7f9315eeda3dd287f8d598f3653d7c107799c5422b3
   Public key (hex): 0xbe5ddb1579b72e84524fc29e78609e3caf42e85aa118ebfe0b0ad404b5bdd25f
   Account ID:       0xbe5ddb1579b72e84524fc29e78609e3caf42e85aa118ebfe0b0ad404b5bdd25f
-  SS58 Address:     15Jbynf3EcRqdHV1K14LXYh7PQFTbp5wiXfrc4kbMReR9KxA
+  SS58 Address:     5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY
 ```
 
 Polkadot grandpa session key for `//Alice` (`ed25519` cryptography).
 
 ```bash
-$ subkey inspect --scheme ed25519 --network polkadot //Alice
+$ subkey inspect --scheme ed25519 --network substrate //Alice
```

Comment:
```suggestion
$ polkadot key inspect --scheme ed25519 --network substrate //Alice
```
These are relay chain values. And `polkadot` now includes the `key` subcommand. If you use it instead, you can save the user from having to build `subkey`